src/loaddb/load_common.cpp

```diff
@@ -344,11 +343,14 @@ namespace cubload
 
 	// scan first string, and ignore rest of the line
 	sscanf (line.c_str (), fmt, class_name.c_str ());
-	std::transform (class_name.begin (), class_name.end (), class_name.begin (), [] (unsigned char c)
-	{
-	  return std::tolower (c);
-	});
-	ignore_classes.emplace_back (class_name.c_str (), strlen (class_name.c_str ()));
+
+	char lower_case_string[DB_MAX_CLASS_LENGTH] = { 0 };
```

Comment:
Please use DB_MAX_IDENTIFIER_LENGTH
It seems DB_MAX_CLASS_LENGTH is just defined :)